example/src/main/scala/example/Example.scala

```diff
@@ -123,14 +123,10 @@ object XMLHttpRequest{
   @JSExport
   def main(pre: html.Pre) = {
     val xhr = new dom.XMLHttpRequest()
-    xhr.open("GET",
-      "http://api.openweathermap.org/" +
-      "data/2.5/weather?q=Singapore"
-    )
+    xhr.open("GET", "https://www.boredapi.com/api/activity")
```

Comment:
These examples show up in the final document next to their output, and hence they only have ~40 characters a line. We should try to keep lines as short as possible:
```suggestion
    xhr.open("GET",
      "https://www.boredapi.com/api/activity")
```